.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: test
+
+on:
```

Comment:
Do we need something else here to trigger on commits too?

Reply:
Note: By default, a workflow only runs when a pull_request's activity type is opened, synchronize, or reopened.

`synchronize` handles new commits.

Reply:
Right, but that's about activity in a pull request.
I'm asking about having it trigger when the PR is merged into master too so every change to master is validated.

Reply:
I think you want to trigger on `push` too for that.

Reply:
Done